server/routes/shared/riskPresenter.ts

```diff
@@ -13,6 +13,10 @@ export default class RiskPresenter {
 
   readonly riskSummaryNotFound = this.riskSummary === null
 
+  get riskInformationAvailable(): boolean {
```

Comment:
I know this is really small, but is it worth adding a test for this? Particularly if it's used by `introText`?

Reply:
yeh i did want to do that, but my stomach was rumbling. i can add now.